index.bs

```diff
@@ -1160,13 +1183,16 @@ The <dfn method for="XRWebGLLayer">requestViewportScaling(|scaleFactor|)</dfn> m
 
   1. If |scaleFactor| is greater than 1.0 set |scaleFactor| to 1.0.
   1. If |scaleFactor| is less than the [=minimum viewport scale factor=] set |scaleFactor| to the [=minimum viewport scale factor=].
-  1. If the [=XR device=] places additional device-specific restrictions on viewport size, adjust |scaleFactor| accordingly.
+  1. If the [=/XR device=] places additional device-specific restrictions on viewport size, adjust |scaleFactor| accordingly.
+
+    Issue: Clarify which XR device the XR device in this step references to.
```

Comment:
This would be [=XR/XR device=]

Reply:
Yes, this shouldn't be [=/XR device=] but needs to be a concrete reference that the step can use. Here, [=XR/XR device=] would still not be entirely a correct handle as we don't have (specification) ways to reference to the XR object's state within this algorithm. Let's get it sorted going forward. For now, I addressed your comment by using [=XR/XR device=], which is closer to what we mean here over [=/XR device=].